src/main/java/org/folio/circulation/domain/representations/ChangeDueDateRequest.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.circulation.domain.representations;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getDateTimeProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.failedValidation;
+
+import org.folio.circulation.support.Result;
+import org.joda.time.DateTime;
+
+import io.vertx.core.json.JsonObject;
+
+public class ChangeDueDateRequest {
+  private static final String DUE_DATE = "dueDate";
+
+  private final String loanId;
+  private final DateTime dueDate;
+
+  public ChangeDueDateRequest(String loanId, DateTime dueDate) {
+    this.loanId = loanId;
+    this.dueDate = dueDate;
+  }
+
+  public static Result<ChangeDueDateRequest> from(JsonObject json,
```

Comment:
Is it used anywhere?

Reply:
Used by `ChangeDueDateResource`, which in turn is used by `CirculationVerticle`, `ChangeDueDateRequestBuilder`, and tests.